src/main/java/com/csc366/project/entity/Owner.java

```diff
@@ -0,0 +1,95 @@
+package com.csc366.project.entity;
+
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+import java.util.StringJoiner;
+
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name="owner")
+public class Owner {
+    @Id
+    @GeneratedValue(strategy=GenerationType.IDENTITY)
+    private Long id;
+
+    private String name;
+
+    private String firstName;
+    private String lastName;
```

Comment:
I'm also confused why we have both name and firstName+lastName, and we only set name in the constructor

Reply:
I left firstName lastName just as an option to split the entire name. I chose to just do "name" since I have it findByName when it is tested. Lmk if you want me to change anything or have suggestions

Reply:
Gotcha makes sense. This is fine for now 👍🏻